packages/grafana-eslint-rules/rules/no-border-radius-literal.cjs

```diff
@@ -0,0 +1,48 @@
+// @ts-check
+const { ESLintUtils, AST_NODE_TYPES } = require('@typescript-eslint/utils');
+
+const createRule = ESLintUtils.RuleCreator((name) => `https://github.com/grafana/grafana#${name}`);
+
+const borderRadiusRule = createRule({
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
+          node.callee.type === AST_NODE_TYPES.Identifier &&
+          node.callee.name === 'css' &&
+          node.arguments[0].type === 'ObjectExpression'
+        ) {
+          for (let property of node.arguments[0].properties) {
```

Comment:
I think `css()` can be called with any number of arguments? I think `css({width: 10}, {height: 10})` is valid?

Either way, I think it is 'cleaner' to loop over all arguments to function calls, rather than hard coding just the first one.